code/modules/stock_market/computer.dm

```diff
@@ -6,6 +6,7 @@
 	icon_keyboard = "no_keyboard"
 	var/logged_in = "Cargo Department"
 	var/vmode = 1
+	var/borrow = 0 //Making it prevent borrowing abuse and stuff
```

Comment:
name it something more descriptive like current_borrow_amounts

You also never decrement this value, so after you have borrowed 2 times you will never be able to borrow again during a round